stmhal/boards/openocd_stm32f4.cfg

```diff
@@ -0,0 +1,42 @@
+# This script configures OpenOCD for use with an ST-Link V2 programmer/debugger
+# and an STM32F4 target microcontroller.
+#
+# To flash your firmware:
+#
+#    $ openocd -f openocd_stm32f4.cfg \
+#        -c "stm_flash build-BOARD/firmware0.bin build-BOARD/firmware1.bin"
+#
+# For a gdb server on port 3333:
+#
+#    $ openocd -f openocd_stm32f4.cfg
+
+
+source [find interface/stlink-v2.cfg]
+transport select hla_swd
+source [find target/stm32f4x.cfg]
+reset_config srst_only
+init
+
+proc stm_flash { BIN0 BIN1 } {
+    reset halt
+    sleep 100
+    wait_halt 2
+    flash write_image erase $BIN0 0x08000000
+    sleep 100
+    verify_image $BIN0 0x08000000
+    sleep 100
+    flash write_image erase $BIN1 0x08020000
+    sleep 100
+    verify_image $BIN1 0x08020000
+    sleep 100
+    reset run
+    shutdown
+}
+
+proc stm_erase {} {
```

Comment:
No. It was in the OpenOCD configuration file for a STM32F0, which I used as a source. I adapted and tested it, and AFAICS it works. But I can remove it, if you want.